src/generatedApiTypes.ts

```diff
@@ -5,7 +5,7 @@
 
 export interface paths {
   '/events/{request_id}': {
-    /** This endpoint allows you to get events with all the information from each activated product - BOTD and Fingerprinting. Use the requestId as a URL path :request_id parameter. This API method is scoped to a request, i.e. all returned information is by requestId. */
+    /** This endpoint allows you to get events with all the information from each activated product (Fingerprint Pro or Bot Detection). Use the requestId as a URL path :request_id parameter. This API method is scoped to a request, i.e. all returned information is by requestId. */
```

Comment:
`and` because you get results from both products in the response, right? (at least in the happy case of no bot detected)

```suggestion
    /** This endpoint allows you to get events with all the information from each activated product (Fingerprint Pro and Bot Detection). Use the requestId as a URL path :request_id parameter. This API method is scoped to a request, i.e. all returned information is by requestId. */
```

Reply:
We need to fix the schema for it https://github.com/fingerprintjs/fingerprint-pro-server-api-openapi